lib/measures/VentilationQAQC/resources/report.html.in

```diff
@@ -15,7 +15,7 @@ $(document).ready(function () {
     $.each(obj, function (index, graphData) {
         var graph = obj[index];
 
-        for (var i = 0; i < graph.timeseries.length; i++) {
+        for (var i = 0; i < 8760; i++) {
```

Comment:
@nllong I'm going to fix this and test, will commit in a few minutes. It should not have changed, I only did that to troubleshoot.

Reply:
Change has been made, CI is running now.